### PR TITLE
Compare the properties of the verifiers in a table.

### PR DESCRIPTION
**Pull request by shuhaowu**
This transcribes an internal document we have comparing the properties of the different verifiers. The intricacies of the verifiers should probably be documented better, but this is a start.

### CODE REVIEW

File: docs/source/verifiers.rst [hunk 1]
```diff
@@ -39,14 +39,67 @@ are given below:
...
+       https://bugs.mysql.com/bug.php?id=87847. This applies to every row in
+       this table.
+
+.. [4] If the rows modified by the the rogue application is modified again on
+       the source after Ghostferry starts, the InlineVerifier's binlog tailer
+       should pick up that row and attempt to reverify it.
+
+.. [5] If the rows missed after resume is modified again on the source after
+       Ghostferry starts, the InlineVerifier's binlog tailer should pick up
+       that row and attempt to reverify it.
```
Comment (fjordan) [inline docs/source/verifiers.rst:92]: good points 👍

File: docs/source/verifiers.rst [hunk 2]
```diff
@@ -39,14 +39,67 @@ are given below:
...
+| binlog coordinates.                               |               |               |                 |
++---------------------------------------------------+---------------+---------------+-----------------+
+| Data inconsistency if Ghostferry's Binlog writing | Yes           | Probably not  | Probably not    |
+| implementation is incorrect and modified the      |               | [6]_          | [6]_            |
+| wrong row on the target (example, an UPDATE is    |               |               |                 |
+| supposed to go to id = 1 but Ghostferry instead   |               |               |                 |
+| issued a query for id = 2). This is an unrealistic|               |               |                 |
+| scenario, but is included for illustrative        |               |               |                 |
+| purposes.                                         |               |               |                 |
++---------------------------------------------------+---------------+---------------+-----------------+
```
Comment (fjordan) [inline docs/source/verifiers.rst:77]: interesting case to think about 🤔  this is if the binlog writer happens to compose and then write an incorrect row due to an incorrect pagination key?

**Review by fjordan — APPROVED** [review]
Thanks for the update. A few questions.

File: docs/source/verifiers.rst (hunk 2 above)
Comment (shuhaowu) [reply docs/source/verifiers.rst:77]: Yes, this would be difficult/impossible to catch with anything other than the ChecksumTable verifier.

**Pull request merged**